src/foam/nanos/auth/AuthenticatedUserDAO.java

```diff
@@ -10,7 +16,12 @@
 import foam.nanos.auth.AuthService;
 import foam.nanos.auth.User;
 import java.security.AccessControlException;
+
+import foam.util.SafetyUtil;
 import net.nanopay.model.Account;
```

Comment:
We need to something about the fact that this class in FOAM2 is referencing a class in nanopay.  Doesn't have to be done in the PR but we need a plan for it soon, right now building nanos outside of nanopay is broken.

Reply:
It's not actually being used in this class so I can just remove it.

Reply:
Did you reply to the right comment?  It looks like Account is used in many places in this class.

Reply:
I did reply to the right comment. It's not being used anywhere here.

Reply:
Oh I see, didn't read your changes closely enough, you removed all the references to Account.